index-test.js

```diff
@@ -0,0 +1,71 @@
+
+require('dotenv').config()
+const request = require('request')
+
+const clientId = process.env.CLIENT_ID
+const idUrl = `https://api.twitch.tv/kraken/users?login=nightblue3&client_id=${clientId}`
+const streamUrl = `https://api.twitch.tv/kraken/streams/26946000?client_id=${clientId}`
+const channelUrl = `https://api.twitch.tv/kraken/channels/26946000?client_id=${clientId}`
+const channelSearchUrl = `https://api.twitch.tv/kraken/search/channels?query=nightblue3&client_id=${clientId}`
+
+const idOptions = {
+  url: idUrl,
+  headers: {
+    'Accept': 'application/vnd.twitchtv.v5+json'
+  },
+  json: true
+}
+
+const streamOptions = {
+  url: streamUrl,
+  headers: {
+    'Accept': 'application/vnd.twitchtv.v5+json'
+  },
+  json: true
+}
+
+const channelOptions = {
+  url: channelUrl,
+  headers: {
+    'Accept': 'application/vnd.twitchtv.v5+json'
+  },
+  json: true
+}
+
+const channelSearchOptions = {
+  url: channelSearchUrl,
+  headers: {
+    'Accept': 'application/vnd.twitchtv.v5+json'
+  },
+  json: true
+}
+
+request(idOptions, (err, res, body) => {
+  console.log(err)
+  console.log(body)
+})
+
+request(streamOptions, (err, res, body) => {
+  console.log(err)
+  console.log(body)
+})
+
+request(channelOptions, (err, res, body) => {
+  console.log(err)
+  console.log(body)
+})
+
+request(channelSearchOptions, (err, res, body) => {
+  console.log(err)
+  console.log(body)
+})
+
+// when stream is not in session, value is null on stream property
+// when stream is in session, value is an object
+
+// channel info gives description, logo, views, followers, display name
+
+// Get id from login
+// use id to get stream and channel info
+
+// channelSearch gives id, display name, logo, description, views, followers, etc
```

Comment:
having a file like this to test is fine, but create a folder called `scratchpad` or something similar. you can place things like notes, design docs, etc.